doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -906,6 +906,14 @@ Quitting and debugging
    time needed to execute it.
 
 
+.. cmd:: Instructions @sentence
```

Comment:
I could not find a place to document command-line flags, so I gather the output of `-help` is considered sufficient.

Reply:
We document some of them in https://coq.inria.fr/refman/practical-tools/coq-commands.html#command-line-options

Reply:
I see that `-time` and `-time-file` are not documented there, so I'll be consistent and not document `-instr` and `-instr-file` there either.

Reply:
I don't think there is a good reason that `-time` isn't included in there. I guess you could try to document both in a follow up PR.